src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -22,6 +22,7 @@
 using Microsoft.ServiceFabric.Actors.Runtime;
 using Microsoft.ServiceFabric.Data;
 using Asset = Maestro.Contracts.Asset;
+using AssetData = Microsoft.DotNet.Maestro.Client.Models.AssetData;
```

Comment:
Was `AssetData` already used or this is a not used `using`.

Reply:
This is to disambiguate with the other AssetData